presto-memsql/src/main/java/io/prestosql/plugin/memsql/MemSqlClient.java

```diff
@@ -51,7 +47,7 @@ public MemSqlClient(BaseJdbcConfig config, @StatsCollecting ConnectionFactory co
             while (resultSet.next()) {
                 String schemaName = resultSet.getString("TABLE_CAT");
                 // skip internal schemas
-                if (!schemaName.equalsIgnoreCase("information_schema") && !schemaName.equalsIgnoreCase("mysql")) {
+                if (!schemaName.equalsIgnoreCase("information_schema") && !schemaName.equalsIgnoreCase("memsql")) {
```

Comment:
memsql has `information_schema` (https://docs.memsql.com/v6.8/concepts/information-schema/)
does it also have a builtin `memsql` schema?

if not, just remove the `&& !schemaName.equalsIgnoreCase("memsql")` part

Reply:
When I tested on memsql docker image, it has builtin `memsql` schema. I think we don't need to remove the condition.

Reply:
@findepi memsql does have schema as `memsql`. The condition needs to be there.

Reply:
@kranthikiran01 @ebyhr thanks for confirming. I couldn't find this in their docs.